rstudio/rstudio
# Fix open link tooltip on Windows

### Intent

Address https://github.com/rstudio/rstudio/issues/12504.

Since Electron, Ctrl+click is used for opening links, not Shift + Click.

### Approach

Change the tooltip in both FR and English for non-mac OS

### Automated Tests

NA

### QA Notes

The tooltip should show Ctrl+Click when hovering over a link. And Ctrl+click should open a link on Linux and Windows

### Documentation

NA

### Checklist


I signed the contributor agreement.
